warehouse/templates/base.html

```diff
@@ -210,9 +210,9 @@ <h3>Get help</h3>
             <h3>About PyPI</h3>
           </li>
           {% if request.registry.settings.get("statuspage.url") %}
-          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">All Systems Operational</span></a></li>
+          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">all systems operational</span></a></li>
```

Comment:
No changes needed here?

Reply:
[Material Design](https://web.archive.org/web/20180410101124/https://material.io/guidelines/style/writing.html#writing-tone) says:

> **Use sentence-style caps**
>
> Titles, headings, labels, and menu items should use sentence-style capitalization (capitalizing only the first word of a sentence).
> 
> Avoid capitalizing all letters except where the Material spec requires them, such as Button text.

And I don't think it's a proper noun, just the current status. But happy to change it if needed, let me know!

Reply:
Here is it in context:

![image](https://user-images.githubusercontent.com/1324225/39966014-2b5a92a4-56ad-11e8-9718-4150c7ba689e.png)

https://pypi.org/

Reply:
I think this change is fine.